docs/development/tutorial/README.md

```diff
@@ -0,0 +1,56 @@
+# NEW DEVELOPER GETTING STARTED!
+
+## Assumptions
+
+- Some familiarity with 'git' is assumed
+- Some familiarity with a favorite
```

Comment:
Broken sentence?

Reply:
Thanks for finding this :+1: 
Yeah, just bad editting here.